pkg/yurtctl/cmd/convert/edgenode.go

```
@@ -269,22 +266,44 @@ func (c *ConvertEdgeNodeOptions) RunConvertEdgeNode() (err error) {
 		if err != nil {
```

how about move `c.KubeadmConfPath` and `c.PodMainfestPath` parameter check before `LabelNode`? so convert can fail fast.

---

It's a good idea.